src/main/java/cn/nukkit/level/generator/biome/ExtremeHillsEdgeBiome.java

```diff
@@ -0,0 +1,28 @@
+package cn.nukkit.level.generator.biome;
+
+import cn.nukkit.level.generator.populator.PopulatorTree;
+
+/**
+ * author: MagicDroidX

```

Comment:
hm?

Reply:
duplicating the class keeps the header lol

Reply:
ok